index.html

```diff
@@ -36,7 +36,158 @@ <h2>
             <div>
                <img src="Images/Social Media.png" alt="Social icons">
             </div>
-        
+        </section>
+        <section class="works">
+            <div class="content1">
+                <h1>
+                    My Recent Works
+                </h1>
+                <img src="Images/Img Placeholder.png" alt="">
+                <h2>
+                    Multi-Post Stories
+                </h2>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s,
+                     when an unknown printer took a standard dummy text.
+                </p>
+                <div>
+                    <h3>css</h3>
+                    <h3>html</h3>
+                    <h3>bootstrap</h3>
+                    <h3>Ruby</h3>
+                </div>
+                <button>See Project</button>
+            </div>
+            </div>
+            <div class="cards">
+                <h1>
+                    Profesional Art Printing Data
+                </h1>
+                <p>
+                    A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required. has been the industry's standard
+                </p>
+                <div>
+                    <h3>html</h3>
+                    <h3>bootstrap</h3>
+                    <h3>Ruby</h3>
+                </div>
+                <button>See Project</button>
+            </div>
+            <div class="cards">
+                <h1>
+                    Profesional Art Printing Data
+                </h1>
+                <p>
+                    A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required. has been the industry's standard
+                </p>
+                <div>
+                    <h3>html</h3>
+                    <h3>bootstrap</h3>
+                    <h3>Ruby</h3>
+                </div>
```

Comment:
- [ ] Kindly I recommend to you use a `ul,` each of them in a `li`list of project technologies (e.g., "JavasScript", "HTML" and "CSS") , This way will be semantically correct